src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -20388,6 +20394,60 @@ public void M()
                 );
         }
 
+        [Fact]
+        public void ConstructorUsesStateFromInitializers()
+        {
+            var source = @"
+public class Program
+{
+    public string Prop { get; set; } = null!;
+
+    public Program()
+    {
+        Prop.ToString();
+    }
+}
+";
+            var comp = CreateNullableCompilation(new[] { source });
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void ConstMembersUseDeclaredNullability()
+        {
+            var source = @"
+public class Program
+{
+    const string s1 = ""hello"";
+    public static readonly string s2 = s1.ToString();
+
+    public Program()
+    {
+    }
+}
+";
+            var comp = CreateNullableCompilation(new[] { source });
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void ComputedPropertiesUseDeclaredNullability()
+        {
+            var source = @"
+public class Program
+{
+    string Prop => ""hello"";
+
+    public Program()
+    {
+        Prop.ToString();
+    }
+}
+";
+            var comp = CreateNullableCompilation(new[] { source });
+            comp.VerifyDiagnostics();
+        }
```

Comment:
>} [](start = 8, length = 1)

Consider moving these new tests outside the block of "MemberNotNull_" tests.

Reply:
Done